specs/pvs/corla.pvs

```diff
@@ -1053,9 +1136,9 @@ BEGIN
 %%% ballot polling audits, but in no event will the risk limit exceed
```

Comment:
Isn't that part of the spec a verbatim reproduction of RDPR-6-Jul-2017 Section 25.2.2(A)?